stroeer/core/v1/article.proto

```diff
@@ -76,3 +89,13 @@ enum AssetType {
   ASSET_TYPE_URL = 3;
   ASSET_TYPE_METADATA = 4;
 }
+
+enum HtmlNodeType {
+  HTML_NODE_TYPE_UNSPECIFIED = 0;
+  HTML_NODE_TYPE_PARAGRAPH = 1;
+  HTML_NODE_TYPE_SUB = 2;
+  HTML_NODE_TYPE_SUP = 3;
+  HTML_NODE_TYPE_ANCHOR = 4;
+  HTML_NODE_TYPE_UNORDERED_LIST = 5;
+  HTML_NODE_TYPE_LIST_ITEM = 6;
```

Comment:
This can be long list if we are to catch every other `HTMLElement`, question is do we really need to do it or just separate the blacklisted tags like `script`, `iframe` and let other pass as safe entities?

Reply:
.. but I get the point that its also clear what kind of elements we support.

Reply:
Its a long list, which means initially we probably have to add and remove things a lot. But this should get stable pretty fast from what we've seen from the previous employee.